app.js

```diff
@@ -42,57 +62,163 @@ app.post('/sync-from-kbo/:kboStructuredIdUuid', async function(req, res) {
-  }
-}, null, true);
+function getKboFields(data) {
+  console.log(data.organisationClassifications);
+
+  let changeTime = data.changeTime;
+  let organisationName = data.name;
+  let ovoNumber = data.ovoNumber;
+  let kboNumber = data.kboNumber;
+
+  //no lables = undefined
+  let formalName = data.labels
+    ?.filter((fields) => {
+      return fields.labelTypeId === "83c1c22a-6776-0ad6-68d2-819e1c6eec66";
+    })
+    .pop()?.value;
+
+  let email = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "f611e91d-a811-4519-afa6-08f0e9dcdd9b" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+
+  let rechtsvorm = data.organisationClassifications
+    ?.filter((fields) => {
+      console.log(fields);
+      return (
+        fields.organisationClassificationTypeId === "9bae7539-64fd-5759-743c-ad9dfa4143d4" ||
+        fields.organisationClassificationTypeId === "1131205e-9212-435d-b4cd-b0d955d08bcf" 
+      );
+    })
+    .pop()?.organisationClassificationName;
+
+    console.log(rechtsvorm)
+
+  let phone = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "d792d4ff-8c34-4336-8434-56ecb40f2fa4" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+
+  let website = data.contacts
+    ?.filter((fields) => {
+      return (
+        fields.contactTypeId === "6763f372-02c5-478c-b7d7-802dc60a64b9" &&
+        !fields.validity.hasOwnProperty("end")
+      );
+    })
+    .pop()?.value;
+
+  let province = data.parents?.parentOrganisationName;
+
+  let formattedAddress = data.locations
+    ?.filter((fields) => {
+      return (
+        fields.isMainLocation === true &&
+        !fields.validity?.hasOwnProperty("end")
+      );
+    })
+    .pop()?.formattedAddress;
+
+  let adressComponent = data.locations
+    ?.filter((fields) => {
+      return (
+        fields.isMainLocation === true &&
+        !fields.validity?.hasOwnProperty("end")
+      );
+    })
+    .pop()?.components;
+   
+
+  return {
+    changeTime: (typeof changeTime != 'undefined' ? changeTime : ""),
+    organisationName: (typeof organisationName != 'undefined' ? organisationName : ""),
+    ovoNumber: (typeof ovoNumber != 'undefined' ? ovoNumber : ""),
+    kboNumber: (typeof kboNumber != 'undefined' ? kboNumber : ""),
+    formalName: (typeof formalName != 'undefined' ? formalName : ""),
+    rechtsvorm: (typeof rechtsvorm != 'undefined' ? rechtsvorm : ""),
+    email: (typeof email != 'undefined' ? email : ""),
+    phone: (typeof phone != 'undefined' ? phone : ""),
+    website: (typeof website != 'undefined' ? website : ""),
+    province: (typeof province != 'undefined' ? province : ""),
+    formattedAddress: (typeof formattedAddress != 'undefined' ? formattedAddress : ""),
+    adressComponent: (typeof adressComponent != 'undefined' ? adressComponent : ""),
```

Comment:
Is there any reason here not to use the nullish coalescing operator `??` instead of the `typeof` in combination with an `if`?